src/Symfony/Component/Validator/Tests/Mapping/Loader/AttributeLoaderTest.php

```diff
@@ -212,7 +213,7 @@ public function testLoadGroupSequenceProviderAnnotation()
 
     protected function createAnnotationLoader(): AnnotationLoader
     {
-        return new AnnotationLoader();
+        return new AttributeLoader();
```

Comment:
It does, `AttributeLoader` extends `AnnotationLoader`. This class has two child classes which override this method. I have to keep the return type wide, otherwise I would break those child classes.